packages/vats/src/lib-chainStorage.js

```diff
@@ -34,7 +34,7 @@ export function makeChainStorageRoot(toStorage, storeName, rootPath) {
         assert.typeof(name, 'string');
         assert(
           pathSegmentPattern.test(name),
-          X`Path segment must be a short ASCII identifier: ${name}`,
+          X`Path segment must be a short snake_case alphanumeric identifier: ${name}`,
```

Comment:
snake_case? I must have missed that; help me find docs for that?

Reply:
https://github.com/Agoric/agoric-sdk/blob/3ce33c1c08ae4fcbff2fa0b087381a8d4be369a7/packages/vats/src/lib-chainStorage.js#L9-L12